softserve/model.py

```diff
@@ -15,6 +15,7 @@ class User(db.Model):
     token = db.Column(db.String(1000))
     email = db.Column(db.String(100), unique=True)
     name = db.Column(db.String(100))
+    pubkey = db.Column(db.String(1024), nullable=False)
```

Comment:
That length is definitely not enough. My pubkey is ~700 characters. If I had two of them, it would easily overflow this.